src/shared/popups/gokb-kbart-import-popup/gokb-kbart-import-popup.vue

```diff
@@ -496,21 +520,20 @@
         try {
           this.options.lineCount = csvDataRows.length - 1
         } catch (exception) {
-          this.error = exception
+          this.errors.push(exception)
         } finally {
           this.completion = 100
           this.importRunning = false
         }
       },
       ensureFieldCounter (colName, severity, type) {
-        if (!this.loadedFile[severity][colName]) {
-          this.loadedFile[severity][colName] = {}
+        if (!this.loadedFile[severity][type]) {
+          this.loadedFile[severity][type] = {}
         }
-
-        if (!this.loadedFile[severity][colName][type]) {
-          this.loadedFile[severity][colName][type] = 1
+        if (!this.loadedFile[severity][type][colName]) {
+          this.loadedFile[severity][type][colName] = 1
         } else {
-          this.loadedFile[severity][colName][type]++
+          this.loadedFile[severity][type][colName]++
```

Comment:
This does not conform to the layout in data: 

```
        loadedFile: {
          columns: [],
          errors: {
            missingColumns: [],
            single: [],
            type: {}
          },
          warnings: {
            single: [],
            type: {}
          },
          lineStats: {
            total: 0,
            warning: 0,
            error: 0
          }
        },
```
We want to create a new key for the object `this.loadedFile.error.type`, for example `this.loadedFile.error.type.publication_type.invalid = 1`. So we want to group these first by column and only then by the type of error.

Reply:
You seem to have a precise idea whereas I don't understand the point. It's easier if you change this, @hornmo . I'm interested in your changes. You can also remove my commit - these changes have been done separately.

Reply:
Thanks, I've seen [2b4681a](https://github.com/openlibraryenvironment/gokb-ui/pull/71/commits/2b4681a1e6fd43baaa4790c2d40b04b22f7f77e9). If this is it, fine.